src/server/routes/supplier_banks.js

```diff
@@ -0,0 +1,48 @@
+const SupplierBank = require('../queries/supplier_banks');
+
+module.exports = function (app, db, config) {
+  SupplierBank.init(db, config).then(() =>
+  {
+    app.get('/api/suppliers/:supplierId/banks', (req, res) => sendSupplierBanks(req, res));
+    app.post('/api/suppliers/:supplierId/banks', (req, res) => createSupplierBank(req, res));
+    app.get('/api/suppliers/:supplierId/banks/:bankAccountId', (req, res) => sendSupplierBank(req, res));
+    app.put('/api/suppliers/:supplierId/banks/:bankAccountId', (req, res) => updateSupplierBank(req, res));
+    app.delete('/api/suppliers/:supplierId/banks/:bankAccountId', (req, res) => deleteSupplierBank(req, res));
+  });
```

Comment:
At least authorization is missing. Currently every supplier would be able to get other suppliers bank details.
Proposed solution is to remove supplierid from url and use implicit from user data. 
By that cross tenant access is already impossible